doc/api.yml

```diff
@@ -1181,6 +1181,115 @@ paths:
           $ref: '#/components/responses/429'
         '500':
           $ref: '#/components/responses/500'
+  '/students/{id}/pointing':
+    post:
+      tags:
+        - Check-in
+      summary: To monitoring the pointing of one student
+      operationId: createPointing
+      parameters:
+        - name: id
+          in: path
+          required: true
+          description: give the student id
+          schema:
+            type: string
+        - name: place
```

Comment:
Why do you use query params for a POST request, why do not you use requestBody ?

Reply:
Changed to request body